### DISCUSSION TITLE
Loading Typescript from <script lang="ts">

### DISCUSSION THREAD

**Discussion by akahanek** [FranckFreiburger/vue3-sfc-loader#110] [General]
Hi,
does the **vue3-sfc-loader** support loading Typescript from <script lang="ts">? Or is it a planned feature?
Thanks, Ales

**Comment by FranckFreiburger** [comment]
[fixed](https://github.com/FranckFreiburger/vue3-sfc-loader/commit/9722616af301819a80eeb8b4e454023d143078c9)
